src/main/java/org/jenkinsci/plugins/runselector/steps/RunSelectorStep.java

```diff
@@ -0,0 +1,98 @@
+package org.jenkinsci.plugins.runselector.steps;
+
+import hudson.Extension;
+import hudson.Util;
+import org.jenkinsci.plugins.runselector.Messages;
+import org.jenkinsci.plugins.runselector.RunFilter;
+import org.jenkinsci.plugins.runselector.RunSelector;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import javax.annotation.CheckForNull;
+
+/**
+ * The runSelector step selects a specific run from a given project name based on the given selector
+ * and, optionally, the run filter.
+ *
+ * @author Alexandru Somai
+ */
+public class RunSelectorStep extends AbstractStepImpl {
+
+    @CheckForNull
+    private final String projectName;
```

Comment:
Both is OK, but `jobName` may be preferred especially for pipeline-plugin.
And it can be simply `job` instead of `jobName` like pipeline-build-step-plugin (https://github.com/jenkinsci/pipeline-build-step-plugin).